cargo-insta/src/cli.rs

```diff
@@ -811,6 +844,12 @@ fn handle_unreferenced_snapshots(
             }
             eprintln!("  {}", path.display());
             if matches!(action, Action::Delete) {
+                let snapshot = Snapshot::from_file(&path)?;
```

Comment:
```suggestion
                let snapshot = match Snapshot::from_file(&path) {
                    Ok(snapshot) => snapshot,
                    Err(e) => {
                        eprintln!("Error loading snapshot at {:?}: {}", &path, e);
                        continue;
                    }
                };
```

Maybe something like that?

Reply:
Yes that's great!